.circleci/config.yml

```diff
@@ -100,35 +102,48 @@ jobs:
       - setup_build_environment
       - snyk/scan:
           additional-arguments: --all-sub-projects --policy-path=.snyk
-  publish:
+  merge-publish:
     executor: java
     steps:
       - setup_build_environment
-      - setup_remote_docker
+      - setup_remote_docker: *latest_remote_docker
       - docker_login
-      - gradle:
-          args: :tag -Prelease
-      - add_ssh_keys:
-          fingerprints:
-            - 'a1:99:97:65:ec:ff:20:cf:4e:7d:92:00:c0:be:63:c4'
-      - run: git push origin $(./gradlew -q :printVersion)
       - gradle:
           args: dockerPushImages
+  release-publish:
+    executor: java
+    steps:
+      - setup_build_environment
+      - setup_remote_docker: *latest_remote_docker
+      - docker_login
+      - gradle:
+          args: publish dockerPushImages
```

Comment:
this project doesn't publish any artifacts, only pushes images

Reply:
so, we should remove `release-publish` flow? or just this publish command?

Reply:
```suggestion
          args: dockerPushImages
```